libs/common-components/src/lib/file-format-select/file-format-select.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="select" [ngClass]="{'single': !isMulti}"
+     (click)="toggle($event)"
+     gdOutside
+     [clickOutsideEnabled]="isOpen"
+     (clickOutside)="onClickOutside($event)">
+  <span class="nav-caret" gdDisabledCursor [dis]="disabled">
+    <fa-icon [ngClass]="{'multi-select': isMulti}" [icon]="['fas','plus']"></fa-icon>
+    <label *ngIf="isMulti">{{label}}</label>
+  </span>
+  <div class="dropdown-menu" *ngIf="isOpen">
+    <div *ngFor="let option of options">
+      <div (click)="select($event, option)" class="option">
+        <fa-icon [icon]="['far', option.icon]"></fa-icon>
```

Comment:
Need to check why sometimes folder is shown as the icon instead of concrete type icon or the `file` icon.